recipes/cjson/all/conanfile.py

```diff
@@ -66,8 +66,7 @@ def _configure_cmake(self):
         self._cmake.definitions["ENABLE_CJSON_TEST"] = False
         self._cmake.definitions["ENABLE_LOCALES"] = self.options.use_locales
         self._cmake.definitions["ENABLE_FUZZING"] = False
-        # Disable Custom Compiler Flags for MingW on Windows, because it uses -fstack-protector-strong
-        self._cmake.definitions["ENABLE_CUSTOM_COMPILER_FLAGS"] = not (self.settings.os == "Windows" and self.settings.compiler == "gcc")
+        self._cmake.definitions["ENABLE_CUSTOM_COMPILER_FLAGS"] = False
```

Comment:
Why we no longer need it?

Reply:
1.7.13 fails to compile on macos with custom flags https://github.com/DaveGamble/cJSON/issues/457, so I decided to unconditionally disable `ENABLE_CUSTOM_COMPILER_FLAGS` instead of adding a new specific case (it enables -Werror which is not a good thing for a package manager, since it can break with new version of compilers).